app/controllers/admin/getting_started_guide_controller.rb

```diff
@@ -19,6 +19,9 @@ def data
     has_sub_path = (path_parts.count == 2 && path_parts[1] != "/")
     sub_path = has_sub_path ? path_parts[1] : ""
 
+    listing_shape = ListingShape.where(community_id: @current_community.id).first
```

Comment:
The other place that used the same expression `ListingShape.where(...).first` wrapped it within a Maybe, should this do that as well? Does this assume a shape is always found for a community? I don't know if anything else is possible, that's why I'm asking.